.eslintrc.js

```diff
@@ -2,10 +2,11 @@ const path = require('path');
 
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   extends: [
     'eslint:recommended',
     'plugin:import/errors',
+    'plugin:import/typescript',
```

Comment:
Allows `eslint-plugin-import` to [parse TypeScript files](https://github.com/benmosher/eslint-plugin-import#typescript)